### PR TITLE
Italian translation of file language.json

### PR DESCRIPTION
**Pull request by bonny1992**
Translated this, too

### CODE REVIEW

File: tr/language.json [hunk 1]
```diff
@@ -3,960 +3,1119 @@
...
 		"DE": "<b>Stunde</b> des Raids auswählen:",
 		"EN": "Select the <b>hour</b> of the raid:",
+		"IT": "Seleziona l'<b>ora</b> del Raid:",
 		"PT-BR": "Select the <b>hour</b> of the raid:"
 	},
 	"raid_select_start_time" : {
 		"NL": "Selecteer de <b>start tijd</b> van de raid:",
 		"DE": "<b>Startzeit</b> des Raids auswählen:",
-		"EN": "Select the <b>start time</b> of the raid:",
+		"IT": "Seleziona l'<b>orario di inizio</b> del Raid:",
```
Comment (Chefkeks) [inline tr/language.json]: Please re-add the English translation ;)

Comment (bonny1992) [reply tr/language.json]: Sure, can you please give me an example of the unnecessary whitespaces/tabs? I seem to can't see them!

Ps: sorry, I didn't notice that I have deleted that translation, must've been a copy/paste mistake!

**Comment by bonny1992** [comment]
If needed I'll make another pull request with the unnecessary whitespaces/tabs, if I understand what's effectively unnecessary.

**Comment by Chefkeks** [comment]
Yes please. A little picture, which illustrates what I mean:

image

**Pull request merged**